demo/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Godot 4+ specific ignores
+.godot/
+/addons/godotopenxr/*/*.aar
+/android/
+*.import
```

Comment:
*.import shouldn't be ignored, these files contain the import settings that determine how a file is imported. 
They are a PITA though,